src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -58,6 +61,10 @@ public async Task<IActionResult> Get()
         [Route("admin/host/extensions/{id}")]
         public async Task<IActionResult> Delete(string id)
         {
+            if (_extensionBundleManager.IsExtensionBundleConfigured())
+            {
+                return BadRequest();
```

Comment:
Update to return BadRequest with an error message: https://docs.microsoft.com/en-us/dotnet/api/system.web.http.apicontroller.badrequest?view=aspnetcore-2.2#System_Web_Http_ApiController_BadRequest_System_String_